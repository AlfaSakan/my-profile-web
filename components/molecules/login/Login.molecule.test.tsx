import renderer from 'react-test-renderer';
import Login from './Login.molecule';

describe('Login component test', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Login />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
