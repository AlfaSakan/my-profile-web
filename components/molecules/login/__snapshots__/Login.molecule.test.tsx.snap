// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component test should render correctly 1`] = `
<div
  className="w-1/2 h-1/2 rounded-2xl flex flex-col overflow-hidden bg-white"
  onClick={[Function]}
>
  <div
    className="flex items-center justify-between px-8 py-4 bg-secondary"
  >
    <p>
      Login / Register
    </p>
  </div>
  <div
    className="flex flex-1 flex-col px-8 py-4 border-y gap-8"
  >
    <div>
      <p>
        Nama
      </p>
      <input
        className="w-full focus:outline-none rounded-md py-2 px-4 border py-3 mt-2"
        placeholder="Type a message"
        type="text"
      />
    </div>
    <div>
      <p>
        Nomor Handphone
      </p>
      <input
        className="w-full focus:outline-none rounded-md py-2 px-4 border py-3 mt-2"
        placeholder="Type a message"
        type="text"
      />
    </div>
  </div>
  <div
    className="flex items-center justify-end px-8 py-4"
  >
    <button
      className="flex items-center justify-center w-32 h-12 rounded-lg bg-secondary"
    >
      <p>
        Login
      </p>
    </button>
  </div>
</div>
`;
